ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobStore.java

```diff
@@ -64,6 +64,10 @@
   private final long minTtlMillis;
   private final boolean requireEncryption;
   private boolean started;
+  // Internal counts for put requests to this partition store since store startup (includes all replicas)
+  private long numPutsRequested = 0, numPutsDone = 0, numPutsSkipped = 0;
+  private long lastTimeLoggedCounts = System.currentTimeMillis();
+  private long logCountsInterval = 60000;
```

Comment:
make this configurable?

Reply:
I may not check in these logging changes.  I'm testing them on one EI host.